ASB Dec 2025 Security Patches integration

alavalas1 opened the pull request:
ASB Dec 2025 Security Patches integration

Integrating Google Android Security Bulletin Patches

Test done: All STS r45 TCs Passed Except 6 TC
CVE_2024_0014 (Issue Id:OAM-133100)
CVE_2016_6786 & Kernel Upstream (Issue Id:OAM-133098) 
CVE_2019_14618& CVE_2016_6786 (Shell offline issue Id: OAM-134000)


Tracked-On: OAM-134611

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
FAILURE: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

alavalas1 committed:
ASB Dec 2025 Security Patches integration

Integrating Google Android Security Bulletin Patches

Test done: All STS r45 TCs Passed Except 6 TC
CVE_2024_0014 (Issue Id:OAM-133100)
CVE_2016_6786 & Kernel Upstream (Issue Id:OAM-133098)
CVE_2019_14618 & CVE_2016_6786 (Shell offline issue Id: OAM-134000)

Tracked-On: OAM-134611
Signed-off-by: alavala srinivasa reddy <alavala.srinivasa.reddy@intel.com>

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

tanujtekriwal reviewed (approved):
LGTM

shyjumon-n reviewed (approved):
LGTM

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is SUCCESS. Please check the linked Tracked-On issue/Android CI Web  for more details. For Binaries: /cactus-absp-or-local/celadon_tmr0_master-merge/122
